src/test/ui/rfc-2565-param-attrs/param-attrs.rs

```diff
@@ -0,0 +1,212 @@
+// compile-pass
+// revisions: bar foo
```

Comment:
It seems that none of the test files account for function pointers at all; at minimum, one of them should.